floor/floor/controller/controller.py

```diff
@@ -49,30 +49,43 @@ def set_fps(self, fps):
         self.fps = fps
         self.frame_seconds = 1.0/fps
 
+    def set_midi_bpm(self, context_value, value):
```

Comment:
It bums me out that python is so picky about its methods being called with exactly the arguments it expects (rather than ignoring extra args) but does not allow polymorphism